styles/global.scss

```diff
@@ -63,3 +65,44 @@ body {
         }
     }
 }
+
+.footer {
+    background: #212121;
+
+    .footer_wrapper {
+        display: flex;
+        justify-content: space-between;
+        padding: 1.5rem 0.3rem;
+        align-items: baseline;
+        max-width: 1240px;
+        margin-inline: auto;
+    }
+    span {
+        font-size: 1rem;
+        display: inline-flex;
+        align-items: center;
+        color: var(--white);
+        i {
+            font-size: inherit;
+            color: var(--red);
+            margin: 0 0.4rem;
+        }
+        a {
+            // text-decoration: underline;
+            // font-weight: 600;
+            margin-left: 0.3rem;
+        }
+    }
+    &__links i {
+        font-size: 1.5rem;
+        margin: 0 0.8rem;
+        color: #b2b2b2;
+        transition: all 0.3s ease 0s;
+        &:hover {
```

Comment:
what is &:hover{ doing?

Reply:
You are checking SCSS file. here it is for hover state of the parent selector.